pandas/tests/resample/test_datetime_index.py

```diff
@@ -641,12 +641,16 @@ def test_resample_dup_index():
         columns=[Period(year=2000, month=i + 1, freq="M") for i in range(12)],
     )
     df.iloc[3, :] = np.nan
-    result = df.resample("Q", axis=1).mean()
-    msg = "DataFrame.groupby with axis=1 is deprecated"
-    with tm.assert_produces_warning(FutureWarning, match=msg):
-        expected = df.groupby(lambda x: int((x.month - 1) / 3), axis=1).mean()
-    expected.columns = [Period(year=2000, quarter=i + 1, freq="Q") for i in range(4)]
-    tm.assert_frame_equal(result, expected)
+    warning_msg = "DataFrame.resample with axis=1 is deprecated."
+    with tm.assert_produces_warning(FutureWarning, match=warning_msg):
+        result = df.resample("Q", axis=1).mean()
+        msg = "DataFrame.groupby with axis=1 is deprecated"
```

Comment:
Could you un-indent the rest of this code?

Reply:
sure, done